Specs/Scene/GltfFeatureMetadataLoaderSpec.js

```diff
@@ -357,17 +357,21 @@ describe(
         ]);
         expect(treesTable.getProperty(1, "species")).toEqual(["Crow"]);
 
-        var colorProperty = mapTexture._properties.color;
-        var intensityProperty = mapTexture._properties.intensity;
-
-        expect(colorProperty._texture.width).toBe(1);
-        expect(colorProperty._texture.height).toBe(1);
-        expect(colorProperty._texture).toBe(intensityProperty._texture);
-
-        var vegetationProperty = orthoTexture._properties.vegetation;
-        expect(colorProperty._texture.width).toBe(1);
-        expect(colorProperty._texture.height).toBe(1);
-        expect(vegetationProperty._texture).not.toBe(colorProperty._texture);
+        var colorProperty = mapTexture.getProperty("color");
+        var intensityProperty = mapTexture.getProperty("intensity");
+        var vegetationProperty = orthoTexture.getProperty("vegetation");
+
+        expect(colorProperty.texture.texture.width).toBe(1);
```

Comment:
Not a big fan of having nested properties with the same name, but I think we have to keep this for symmetry.

Reply:
Maybe `ModelComponents.Texture` could be renamed to `ModelComponents.TextureReader` or something like that?

Reply:
Or `TextureAccessor` to be in line with [textureAccessor.schema.json](https://github.com/CesiumGS/glTF/blob/3d-tiles-next/extensions/2.0/Vendor/EXT_feature_metadata/1.0.0/schema/textureAccessor.schema.json)

Reply:
We talked about this, we decided on `TextureReader`